packages/rule-components/src/ReportDetails/RiskDescription.js

```diff
@@ -7,8 +7,8 @@ const RiskDescription = ({ riskValue, riskMeta }) => {
     const { IconComponent, label, description } = riskMeta[riskValue - 1];
 
     return (
-        <div>
-            <div className="battery">
+        <div className="ins-c-rule__risk-description">
+            <div className={ `battery severity-level-${ riskValue }` }>
```

Comment:
Can you change the name of these 2 classnames to
* `ins-c-rule__battery`
* `ins-c-rule__severity-level-${riskValue}`

Reply:
fixed